ImageInLib/ImageInLib/src/segmentation3D_subsurf.c

```diff
@@ -170,11 +170,12 @@ bool subsurfSegmentation(Image_Data inputImageData, dataType** initialSegment, S
 			sprintf_s(name_ending, sizeof(name_ending), "_seg_func_%03zd.raw", i);
 			strcat_s(name, sizeof(name), name_ending);
 			store3dDataArrayD(imageData.segmentationFuntionPtr, length, width, height, name, flags);
+			printf("Step is %zd\n", segParameters.numberOfTimeStep);
 		}
 		i++;
 	} while ((i <= segParameters.maxNoOfTimeSteps) && (difference_btw_current_and_previous_sol > segParameters.segTolerance));
 
-	printf("finish: Segmentation tolerance is %lf\n", segParameters.segTolerance);
+	//printf("finish: Segmentation tolerance is %lf\n", segParameters.segTolerance);
```

Comment:
if you need this output for debugging purposes, please "control it" by defined constants